vsts/internal-version-increment.sh

```diff
@@ -75,3 +75,8 @@ fi
 if [ "$BUILD_SOURCEBRANCHNAME" != "master" ]; then
   sed -i '' 's/\(VERSION_STRING[[:space:]]*=[[:space:]]*\).*/\1'$publish_version'/g' $VERSION_FILENAME
 fi
+
+# Update swift package manager config file
+swift_package_file='Package.swift'
+sed -i '' 's/\(define("APP_CENTER_C_VERSION",[[:space:]]*to:*\).*/\1''"\\"'$publish_version'\\""),''/g' $swift_package_file
```

Comment:
As mentioned above, if you use something like "SWIFT_PACKAGE_VERSION" instead of a real version, you can simplify this line.
```
.define("APP_CENTER_C_VERSION", to:"\"SWIFT_PACKAGE_VERSION\""),
```
```suggestion
sed -i '' 's/SWIFT_PACKAGE_VERSION/'$publish_version'/g' $swift_package_file
```

Reply:
No, the real version must be stored in this file